scripts/debug_app.sh

```
@@ -0,0 +1,67 @@
+	chmod +x "${PROJECT_ROOT}/gradlew"
+fi
+
+TIMESTAMP=$(date +"%Y%m%d_%H%M%S")
+BUILD_LOG="${LOG_DIR}/assembleDebug_${TIMESTAMP}.log"
+
+echo "🧹 Cleaning..."
+./gradlew clean --no-daemon --quiet || true
+
+echo "🔧 Attempting assembleDebug (this may fail, expected for triage)"
+set +e
+./gradlew :$(basename "${MODULE_DIR}"):assembleDebug --no-daemon --info --stacktrace 2>&1 | tee "${BUILD_LOG}"
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)